alerting/src/main/kotlin/org/opensearch/alerting/alerts/AlertIndices.kt

```diff
@@ -102,12 +126,16 @@ class AlertIndices(
     }
 
     @Volatile private var historyEnabled = AlertingSettings.ALERT_HISTORY_ENABLED.get(settings)
+    @Volatile private var findingEnabled = AlertingSettings.ALERT_FINDING_ENABLED.get(settings)
 
     @Volatile private var historyMaxDocs = AlertingSettings.ALERT_HISTORY_MAX_DOCS.get(settings)
+    @Volatile private var findingMaxDocs = AlertingSettings.ALERT_FINDING_MAX_DOCS.get(settings)
 
     @Volatile private var historyMaxAge = AlertingSettings.ALERT_HISTORY_INDEX_MAX_AGE.get(settings)
+    @Volatile private var findingMaxAge = AlertingSettings.ALERT_HISTORY_FINDING_MAX_AGE.get(settings)
 
     @Volatile private var historyRolloverPeriod = AlertingSettings.ALERT_HISTORY_ROLLOVER_PERIOD.get(settings)
```

Comment:
Rename the original history settings to prepend `alert`. For example, `historyEnabled` will be `alertHistoryEnabled`